voluncheer/chatroom/test_views.py

```diff
@@ -1,27 +1,28 @@
-import random
-import string
-
 from django.contrib.auth import get_user_model
 from django.test import TestCase
 from django.urls import reverse
 
 from profiles.models import UserType
+from profiles.models import Volunteer
 
 
 class ChatroomViewTest(TestCase):
     def setUp(self):
         super().setUp()
-        self.password = "".join(random.choices(string.ascii_uppercase + string.digits, k=8))
         self.user = get_user_model().objects.create_user(
             email="test@voluncheer.com",
             password="secret",
             type=UserType.VOLUNTEER,
         )
         self.user.save()
         self.client.login(email="test@voluncheer.com", password="secret")
-
-    def tearDown(self):
-        self.user.delete()
+        self.vol2 = Volunteer.objects.create(
```

Comment:
So where exactly do you use vol2?

Reply:
I see this is to resolve the bug yesterday right?

Reply:
Yeah. It's not a good idea to add photo attribute to unittest setup. So I add this back.